packages/flutter_tools/lib/src/debug_adapters/README.md

```diff
@@ -1,6 +1,6 @@
 # Debug Adapter Protocol (DAP)
 
-This document is Flutter-specific. For information on the standard Dart DAP implementation, [see this document](https://github.com/dart-lang/sdk/blob/main/pkg/dds/tool/dap/README.md).
+This document is Flutter-specific. For information on the standard Dart DAP implementation, [see this document](https://github.com/dart-lang/sdk/tree/main/pkg/dds/lib/src/dap).
```

Comment:
This new link links to source code, not a document about the standard Dart DAP implementation.

@bkonyi or @DanTup Do you know where the document has moved to that was originally referenced here? Or have a suggestion for how to update this?

Reply:
I'm not sure myself. Maybe Danny will know?

Reply:
Some DAP bits from https://github.com/dart-lang/sdk/blob/main/party/pkg/dds/ moved to https://github.com/dart-lang/sdk/blob/main/third_party/pkg/dap/

So that readme is now at https://github.com/dart-lang/sdk/blob/main/third_party/pkg/dap/tool/README.md.

Reply:
```suggestion
This document is Flutter-specific. For information on the standard Dart DAP implementation, [see this document](https://github.com/dart-lang/sdk/blob/main/third_party/pkg/dap/tool/README.md).
```